README.md

```diff
@@ -6,4 +6,14 @@ A code exercise for refactoring legacy code
 2. [Install JDK 15.0.2](https://www.oracle.com/java/technologies/javase-jdk15-downloads.html)
 3. Clone repository by typing the following in the terminal: git clone https://github.com/kudukacar/GildedRoseKata.git
 4. In your terminal, navigate to the project folder
-5. Type the following in the terminal to run the tests: ./gradlew test
\ No newline at end of file
+5. Run `gradle wrapper --gradle-version 6.3` in the console
```

Comment:
**question** What does `gradle wrapper` do? Does it generate the wrapper script that is included in the repository?

Reply:
We were getting a version error saying that spring boot needs gradle version 6.3 or higher.  When I ran this command it miraculously removed the error.

Reply:
My guess is that you ran this, and it updated the value in `gradle-wrapper.properties` to `6.8.x`.

My understanding is that the point of using the wrapper script `gradlew` is that you don't need to have gradle installed on the system as a prerequisite. Thus, you can probably remove this from the instructions because you're repository already includes the wrapper script that this step is generating.

Reply:
In case it is useful https://docs.gradle.org/current/userguide/gradle_wrapper.html

Reply:
Sounds good